src/main/python/server.py

```diff
@@ -16,15 +18,43 @@
 
 
 class Service(service_pb2_grpc.FeatureExtractorServicer):
-    pass
+    """Feature Extractor Service"""
+
+    def Identifiers(self, request, context):
+        """Extract identifiers weighted set"""
+
+        extractor = IdentifiersBagExtractor(
+            docfreq_threshold=request.docfreqThreshold, split_stem=request.splitStem)
+
+        return self._create_response(extractor.extract(request.uast))
+
+    def Literals(self, request, context):
+        """Extract literals weighted set"""
+
+        extractor = LiteralsBagExtractor(
+            docfreq_threshold=request.docfreqThreshold)
+
+        return self._create_response(extractor.extract(request.uast))
+
+    def Uast2seq(self, request, context):
+        """Extract uast2seq weighted set"""
+
+        extractor = UastSeqBagExtractor(
+            docfreq_threshold=request.docfreqThreshold)
```

Comment:
Same situation with arguments of the base algorithm class [here](https://github.com/src-d/ml/blob/master/sourced/ml/algorithms/uast_ids_to_bag.py#L21) , allthough these guys `token2index` and `token_parser` class name might be of less value right now and could be added later, as soon as they are really needed.

What do you think?

Reply:
I found only 1 implementation for token parser and token index in ml lib. So I decided, for now, we don't need additional parameters there.